### PR TITLE
[9.0][ADD]sale operating unit

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
# Sales with Operating Units

This module introduces the following features:
- Adds the Operating Unit (OU) to sales orders
- Adds rules to access sales order according user’s OU
- Adds retrictions to avoid creating sales orders where the company is different to the OU
- The OU in the sales order must match the OU in the Warehouse
- By default the main OU is related to a stock location
- By default the default OU is added to the sale order

### CODE REVIEW

File: sale_operating_unit/README.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+Installation
+============
+
+No additional installation instructions are required.
+
+Configuration
+=============
+
+Go to 'Settings / Technical / Actions / User-defined Defaults' and remove
```
Comment (JordiBForgeFlow) [inline sale_operating_unit/README.rst]: Remove this, as it is not applicable anymore in v9

File: sale_operating_unit/README.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+the default set for the Shop.
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/213/7.0
```
Comment (JordiBForgeFlow) [inline sale_operating_unit/README.rst]: Change to 9.0

File: sale_operating_unit/README.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+
+* Eficent <contact@eficent.com>
```
Comment (JordiBForgeFlow) [inline sale_operating_unit/README.rst]: Change to the actual contributors:
- Jordi Ballester Alomar jordi.ballester@eficent.com
- Aarón Henríquez aheficent@eficent.com
- Sudhir Arya sudhir.arya@serpentcs.com

**Review by JordiBForgeFlow — APPROVED** [review]
:+1:

**Comment by guewen** [comment]
@aheficent same here https://github.com/Eficent/operating-unit/pull/65

**Pull request merged**